docs/algos/performances.md

```diff
@@ -0,0 +1,43 @@
+# Performance assessments
+
+:warning: This document is a work in progress.
+
+## Introduction
+To ensure the implementation of the algorithms are correct, we want to test them on various environments. For the sake of reproducibility, we want to run for 10 seeds on various environments. For maintenance purposes and long-term support, these tests will be conducted on environments available in [MO-Gymnasium](www.github.com/farama-foundation/mo-gymnasium). Hence, we will not be able to test on some environments that were presented in original papers.
+
+
+## Metrics
+For single-policy algorithms, the metric used will be the scalarized return of the policy.
+
+For multi-policy algorithms, we propose to rely on various metrics to assess the quality of the Pareto Fronts (PF). In general, we want to have a metric that is able to assess the convergence of the PF, a metric that is able to assess the diversity of the PF, and a hybrid metric assessing both. The metrics are implemented in `common/performance_indicators`. We propose to use the following metrics:
+* (Diversity) Sparsity: average distance between each consecutive point in the PF. From the PGMORL paper [1].
+* (Convergence) IGD: a SOTA metric from Multi-Objective Optimization (MOO) literature. It requires a reference PF that we can compute a posteriori. That is, we do a merge of all the PFs found by the method and compute the IGD with respect to this reference PF.
+* (Hybrid) Hypervolume: a SOTA metric from MOO and MORL literature.
+
+Moreover, some metrics relying on assumptions on the utility function of the user are proposed in the literature. These metric allow to have an idea on the true value on the user utility, whereas others such as hypervolume do not. We propose to use the following metrics:
+* EUM: Expected Utility Metric. From [2].
```

Comment:
I suggest also adding Maximum Utility loss (MUL) for the problems we know the true CCS/PF.